src/client/public/scripts/pages/github-app-config.ts

```diff
@@ -0,0 +1,202 @@
+import {html} from '../../../../../node_modules/lit-html/lib/lit-extended.js';
+import {TemplateResult} from '../../../../../node_modules/lit-html/lit-html.js';
+import * as api from '../../../../types/api.js';
+import {BaseElement, property} from '../components/base-element.js';
+
+const DEFAULT_CONFIG = `{
+  // Add custom configuration here
+}`;
+
+class GithubAppConfig extends BaseElement {
+  private orgName: string|null;
+  @property() private errorMsg: string|null;
+  @property() private currentConfig: string|null;
+  @property() private initialised: boolean;
+
+  constructor() {
+    super();
+    this.orgName = null;
+    this.errorMsg = null;
+    this.currentConfig = null;
+    this.initialised = false;
+  }
+
+  async connectedCallback() {
+    this.getLatestConfig();
+  }
+
+  private async getLatestConfig() {
+    const queryParams = new URLSearchParams(window.location.search);
+    let orgName: string|null = queryParams.get('org_name');
+    if (!orgName) {
+      const installId = queryParams.get('installation_id');
+      if (!installId) {
+        this.errorMsg = 'No Install ID defined.';
+        return;
+      }
+      orgName = await this.getOrgNameFromInstallId(installId);
+    }
+
+    if (!orgName) {
+      this.errorMsg = 'No org ID could be found.';
+      return;
+    }
+
+    this.orgName = orgName;
+
+    this.currentConfig = await this.getConfig();
+    this.initialised = true;
+  }
+
+  async getConfig(): Promise<string|null> {
+    const response = await fetch('/api/github-app/get-config', {
+      method: 'POST',
+      headers: {'Content-Type': 'application/json'},
+      body: JSON.stringify({
+        orgName: this.orgName,
+      }),
```

Comment:
It seems like this should be GET.

`GET /api/org/config/:orgname`.

Then to modify it:

`POST /api/org/config/:orgname`.